src/loaders/lottie/tvgLottieLoader.cpp

```diff
@@ -356,8 +356,7 @@ float LottieLoader::duration()
     if (!comp) done();
     if (!comp) return 0.0f;
 
-    auto frameNo = frameCnt * (segmentEnd - segmentBegin);
-    return frameNo / comp->frameRate;
+    return frameCnt * (segmentEnd - segmentBegin) / comp->frameRate;
```

Comment:
I'd put all into one line, but up to you
```return !comp ? 0.0f : frameCnt * (segmentEnd - segmentBegin) / comp->frameRate;```

also in l.371 I'd write:
```return !comp ? 0 :  comp->markers.count;```

Reply:
cleaning code is good but not mandatory. 
This kind of requirement would require endless code changes.
Let's review the diff only. Thanks.